pkg/cfg/binaryinfo.go

```diff
@@ -0,0 +1,66 @@
+package cfg
+
+import (
+	"bytes"
+	"os/exec"
+	"runtime"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Version   - Binary version
+// GitCommit - Commit SHA of the source code
+// BuildDate - Binary build date
+// BuildOS   - Operating System used to build binary
+//nolint:gochecknoglobals
+var (
+	Version   = "unspecified"
+	GitCommit = "unspecified"
+	BuildDate = "unspecified"
+	BuildOS   = runtime.GOOS
+)
+
+//nolint:gochecknoinits
+func init() {
+	info, err := latestGitReleaseInfo()
+	if err != nil {
+		logrus.Errorf("Get latest git release info error: %s\n", err.Error())
+		return
+	}
+
+	Version = info[0]
+	GitCommit = info[1]
+	BuildDate = info[2]
+}
```

Comment:
Not entirely sure but probably not a good idea. Maybe some sort of a getter can be called? Maybe `GetInfo` that loads all the data and then let `Execute` call it and assigns the data. Are the global variables used anywhere else?